### PR TITLE
ci: use the specified version of `skywalking`.

### LINKED ISSUE #3696: ci failed: should use the `skywalking` with the specified version
"Unable to find image 'apache/skywalking-oap-server:latest' locally
docker: Error response from daemon: manifest for apache/skywalking-oap-server:latest not found: manifest unknown: manifest unknown"

https://github.com/apache/apisix/runs/1993910896#step:8:306

related issue: https://github.com/apache/skywalking/issues/6466

### PR DESCRIPTION
**Pull request by membphis**
### What this PR does / why we need it:

fix #3696

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/skywalking.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ plugin_attr:
         3. Run skywalking server:
 
             ```shell
-            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server
+            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server:8.3.0-es7
```
Comment (spacewander) [inline docs/en/latest/plugins/skywalking.md:136]: Should be `8.3.0-es6`?

File: docs/zh/latest/plugins/skywalking.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ plugin_attr:
         3. 启动 skywalking：
 
             ```shell
-            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server
+            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server:8.3.0-es7
```
Comment (spacewander) [inline docs/zh/latest/plugins/skywalking.md:139]: ditto

**Review by spacewander — CHANGES_REQUESTED** [review]

File: docs/en/latest/plugins/skywalking.md (hunk 1 above)
Comment (membphis) [reply docs/en/latest/plugins/skywalking.md:136]: I think there should be `8.3.0-es7`.

I got it from https://github.com/apache/skywalking-docker/blob/master/oap-server/README.md

Or we can remove the doc about `Of Course, you can use Elasticsearch storage`, just link to the doc of SkyWalking.

what do you think @moonming ?

Comment (membphis) [reply docs/en/latest/plugins/skywalking.md:136]: yes, you are right. update it later

Comment (membphis) [reply docs/en/latest/plugins/skywalking.md]: fixed

**Review by juzhiyuan — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**